src/jupyter_contrib_nbextensions/nbextensions/toc2/main.css

```diff
@@ -145,9 +149,18 @@ padding-left: 20px;
 
 /* don't waste so much screen space... */
 
-.toc .toc-item .toc-item {
+/*.toc .toc-item .toc-item {
   padding-left: 1em;
 }
+*/
+.toc-item ul, li { margin:0; padding:0 }
+.toc-item li > span { display:block }
+.toc-item li > span { padding-left:0em }
+.toc-item li li > span { padding-left:1em }
+.toc-item li li li > span { padding-left:2em }
+.toc-item li li li li > span { padding-left:3em }
+.toc-item li li li li li > span { padding-left:4em }
+.toc-item li li li li li li > span { padding-left:5em }
```

Comment:
Why are we adding extra padding, and why separately for various nesting levels?! If desired (is the regular indenting not sufficient?), could this not be handled by adjusting the `ul`'s padding?

Also note that this has set margin & padding for *all* `li` elements to 0 - I guess that was unintentional, and should have been `.toc-item li`?